independent-publisher-connectors/Line Message/apiDefinition.swagger.json

```diff
@@ -0,0 +1,159 @@
+                          "location"
+                        ]
+                      },
+                      "text": {
+                        "type": "string",
+                        "title": "Text Type",
+                        "description": "hello world"
+                      },
+                      "packageId": {
+                        "type": "string",
+                        "title": "Sticker Type",
+                        "description": "Ex:446"
+                      },
+                      "stickerId": {
+                        "type": "string",
+                        "title": "Sticker Type",
+                        "description": "Ex:1988"
+                      },
+                      "originalContentUrl": {
+                        "type": "string",
+                        "title": "Image Type",
+                        "description": "https://example.com/image.jpg"
+                      },
+                      "previewImageUrl": {
+                        "type": "string",
+                        "title": "Image Type",
+                        "description": "https://example.com/image.jpg"
+                      },
+                      "title": {
+                        "type": "string",
+                        "title": "Location Type",
+                        "description": "Ex:My location"
+                      },
+                      "address": {
+                        "type": "string",
+                        "title": "Location Type",
+                        "description": "Ex:My location"
+                      },
+                      "latitude": {
+                        "type": "string",
+                        "title": "Location Type",
+                        "description": "Ex:35.687574,"
+                      },
+                      "longitude": {
+                        "type": "string",
+                        "title": "Location Type",
+                        "description": "Ex:139.72922,"
+                      }
+                    }
+                  },
+                  "description": "messages"
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "Channel Access Token": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "Authorization"
+    }
+  },
+  "security": [
+    {
+      "Channel Access Token": []
+    }
+  ],
+  "tags": [],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://developers.line.biz/en/services/messaging-api/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://developers.line.biz/en/services/messaging-api/"
```

Comment:
@felaray can you please provide a valid URL for the privacy policy page? Maybe "https://developers.line.biz/en/terms-and-policies/"

Reply:
@Amjed-Ayoub I was renew the url.